src/BookForModal.js

```diff
@@ -0,0 +1,37 @@
+import React, { Component } from "react";
+
+function Example() {
+  const [show, setShow] = useState(false);
+
+  const handleClose = () => setShow(false);
+  const handleShow = () => setShow(true);
+}
+
+export default class BookForModal extends Component {
```

Comment:
- There is a typo here on Component, should be `BookFormModal`.
- Once the Modal is ready, we can use it. I don't see it being used quite yet. Grab a TA if you can to get this up and running 👍